tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/expressions/binaryoperations/IntegerRangeOperatorTest.java

```diff
@@ -34,11 +34,14 @@
 public class IntegerRangeOperatorTest {
 
     private CompileResult result;
+    private CompileResult runtimeResult;
     private CompileResult negativeResult;
 
     @BeforeClass
     public void setup() {
         result = BCompileUtil.compile("test-src/expressions/binaryoperations/integer_range_operators.bal");
+        runtimeResult = BCompileUtil.
+                compile("test-src/expressions/binaryoperations/integer-range-operators-runtime.bal");
```

Comment:
We can probably have a separate package for the type hash related tests, right?

How do we test the TypeHashVisitor btw? If not unit tests, I think we need to have at least tests with multiple anon types that may or may not result in the same hash. Shall we create an issue for this?

Reply:
Created https://github.com/ballerina-platform/ballerina-lang/issues/30849